tests/xtro-sharpie/ios.pending

```diff
@@ -46,6 +46,16 @@
 ## fixed for XAMCORE_4_0
 !incorrect-protocol-member! QLPreviewItem::previewItemTitle is OPTIONAL and should NOT be abstract
 
+# UserNotifications
+
+## Shorter enum names
+!missing-enum! UNNotificationContentExtensionMediaPlayPauseButtonType not bound
+!missing-enum! UNNotificationContentExtensionResponseOption not bound
+!unknown-native-enum! UNResponseOption bound
+!unknown-native-enum! UNMediaPlayPauseButtonType bound
+
+## Field to Enum manual mapping
+!unknown-native-enum! UNActionIdentifier bound
```

Comment:
if it's a Xamarin helper enum then it should not be `[Native]` (no need) and that should remove this entry from xtro results